client/src/components/QuestionsAndAnswers.jsx

```diff
@@ -21,7 +23,25 @@ const QuestionsAndAnswers = () => {
 
       {/* NOTE - Refactor to have a seperate component later on when there is more to display */}
       <div className="qa-card-sample">
-        <b><div className="qa-div">Q: Question [Placeholder]</div></b><p></p>
+        <b><div className="qa-div">Q: Question [Placeholder]
+          <button onClick={() => setModalIsOpenAdd(true)} className="qa-add-answer" href="">Add Answer</button>
+          <Modal isOpen={modalIsOpenAdd} onRequestClose={() => setModalIsOpenAdd(false)}>
+            <h2>Submit Your Answer</h2>
+            <h3>[Product Name]: [Question Body]</h3>
+            <form className="add-answer-form">
+              <label className="add-form-answer">Answer*</label>
+              <textarea rows="10" maxLength="1000" required></textarea><p></p>
+              <label className="add-form-username">Nickname*</label>
+              <input maxLength="60" placeholder="Example: jack543!" required></input><p className="form-small">For privacy reasons, do not use your full name or email address</p>
+              <label className="add-form-email">Email*</label>
+              <input maxLength="60" placeholder="Example: jack@email.com" required></input><p className="form-small">For authentication reasons, you will not be emailed</p>
```

Comment:
You can just have the end of an <input maxLength... /> tag with the internal forward slash to complete the statement.

Reply:
Not sure I understand what you mean here, can you give me an example?

Reply:
Sorry, I thought I put an example, so for line 37 this would work: <input maxLength="60" placeholder="Example: jack@email.com" required />

Reply:
Ahh, sorry. I put a code snippet without back-ticks and it disappeared so you didn't see what I was referencing!